tests/rules/null-coallesce.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+- id: Throw
+  languages:
+    - php
+  message: "throw"
+  pattern: |
+    $BB ?? $AA
+  severity: ERROR
+- id: embedded_word
```

Comment:
This rule `embedded_word` is not tested, the `id` much match the comment in the code being tested.

Reply:
`id: test`